meta/generate/generate.go

```diff
@@ -25,9 +25,7 @@ package meta
 {{range $k, $v := .Fields -}}
 // {{title $k}} is a helper function to return a slice copy of {{$v.Key}} values.
```

Comment:
question: this is not longer accurate right? `return a slice copy`

Reply:
good catch, missed that update.